docs/src/js/components/ElementsTextArea.react.js

```diff
@@ -492,6 +506,20 @@ export default class ElementsTextArea extends React.Component {
                     {autoHeightSample}
                 </Highlighter>
 
+                {/* Auto Focus */}
+                <Header size="large" style={{ marginTop: '55px' }} sub={true}>
```

Comment:
Let's remove `={true}` from the `sub` prop.

Reply:
So just pass `sub`?